### PR TITLE
Issue #6749: aligned javadoc/xdoc for WhitespaceAround

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6749 (part of #5750)

No changes in the code.

### CODE REVIEW

**Comment by rnveach** [comment]
pitest failure:
> `-CheckstyleAntTask.java.html:<td class='covered'><pre><span  class='survived'>            if (toFile == null || !useFile) {</span></pre></td></tr>`

I think we have had this before. The test must not be stable for some reason. I will try restarting.

**Comment by rnveach** [comment]
I thought I could re-run it but for some reason it doesn't seem to be letting me.

**Comment by romani** [comment]
from CircleCI:
> Jobs pbludov checkstyle issue-6749-ws-around 571 (pitest1) Rerun workflow

For some reason build is executed in non-organization CI, but in personal account.
@pbludov , please do some code changes, to trigger new build, it could be issue on CI side.

File: src/xdocs/config_whitespace.xml [hunk 1]
```diff
@@ -2437,49 +2437,49 @@ try {
...
-            <td>allow empty catch bodies</td>
+            <td>Allow empty catch bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>7.6</td>
           </tr>
           <tr>
             <td>ignoreEnhancedForColon</td>
-            <td>ignore whitespace around colon in for-each loops</td>
+            <td>Ignore whitespace around colon in enhanced for loops.</td>
```
Comment (romani) [inline src/xdocs/config_whitespace.xml]: @rnveach , 

we need your native look here.
In documentation https://docs.oracle.com/javase/specs/jls/se11/html/jls-14.html#jls-14.14.2 "enhanced for" is used but `for` is in special formatting so it does not mean english word "for". 

I would rather use "enhanced-for loops" or `enhanced <i>for</i> loops` or `<a href="https://docs.oracle.com/javase/specs/jls/se11/html/jls-14.html#jls-14.14.2">enhanced for</a> loop` to be as as close as possible to Jdk specification ( refactoring from "loop" to "statement" is too much ).

Comment (rnveach) [reply src/xdocs/config_whitespace.xml]: The JLS does `<code>for</code>`. I am fine with doing that in addition to using the `a href`.

Comment (pbludov) [reply src/xdocs/config_whitespace.xml]: done.

**Pull request merged**